modules/021-cni-cilium/templates/agent/daemonset.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
   {{- include "helm_lib_module_labels" (list . (dict "app" "agent" "workload-resource-policy.deckhouse.io" "every-node")) | nindent 2 }}
 spec:
   {{- include "helm_lib_resources_management_vpa_spec"  (list "apps/v1" "DaemonSet" "agent" "cilium-agent" .Values.cniCilium.resourcesManagement ) | nindent 2}}
-  {{- if .Values.cniCilium.resourcesManagement.vpa }}
+  {{- if (eq .Values.cniCilium.resourcesManagement.mode "VPA") }}
```

Comment:
How to Static mode should work in this case ?

Reply:
The `helm_lib_vpa_kube_rbac_proxy_resources` doesn't support Static mode.

Reply:
And there will be standard resources spec for container `kube-rbac-proxy` provided by `helm_lib_container_kube_rbac_proxy_resources`:

```
cpu: 10m
memory: 25Mi
```
The issue: https://github.com/deckhouse/deckhouse/issues/6618